### PR TITLE
CVS-78730-client_demos_readme+files

### PR DESCRIPTION
**Pull request by ngrozae**
New created from cvs-78730-new_documentation_test branch.
Changes only for client/ and demos/ readme files.

commit 29857b236b13e9460f88f9dfac0cb7887aca34b8
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Tue Jul 12 09:47:28 2022 +0200

    docs/developer_guide.md updated

commit c5b3ef9dadd2c5192225fceb30a4abcff6f93f23
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Thu Jul 7 07:53:21 2022 +0200

    docs/docker_container.md updated

commit 8e2d8240a8988316969e477ddf44ce1e60c6a848
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Mon Jun 27 14:44:55 2022 +0200

    demos/image_classification/go/README.md updated

commit 6a9fb0da17c70e4133ebc890936f93347057c8e6
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Mon Jun 27 12:10:16 2022 +0200

    demos/face_detection/python/README.md updated

commit e91ef144018f81cde21bb1ff520d9bf8027231c5
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Fri Jun 24 15:15:34 2022 +0200

    demos/benchmark/cpp/README.md demos/image_classification/cpp/README.md demos/person_vehicle_bike_detection/python/README.md docs/developer_guide.md updated

commit c76a61bd20e7e3bcc0de7741d618c74193c294f6
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Thu Jun 23 17:15:43 2022 +0200

    docs/developer_guide.md updated

commit cab0b8185ab91b9018788a491facb490ce68707b
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Thu Jun 23 17:15:01 2022 +0200

    demos/person_vehicle_bike_detection/python/README.md updated

commit bd44250a3bd62aa2ad41017911bcf82f60fe7471
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Thu Jun 23 17:14:36 2022 +0200

    demos/bert_question_answering/python/README.md updated

commit a591fd8352f9fab1e73c810d35b05fe1ab4bd0f9
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Thu Jun 23 09:35:30 2022 +0200

    demos/optical_character_recognition/python/README.md updated

commit 62f2b03fe618acabf126a1420532326c2ede1d1f
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Thu Jun 23 09:32:34 2022 +0200

    demos/real_time_stream_analysis/python/use_cases/person_vehicle_bike_detection/README.md updated

commit 30634e3d1849cd5851cf8db26beccc43579bd592
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Thu Jun 23 09:31:56 2022 +0200

    demos/real_time_stream_analysis/python/README.md updated

commit 740a4b23744dd521b1cab1913e1e3b0f2f9d0c74
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Thu Jun 23 09:26:31 2022 +0200

    demos/speech_recognition_with_kaldi_model/python/README.md updated

commit 9a0741e42770ff32a9256add05fffa0c073ecb14
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Tue Jun 21 14:31:03 2022 +0200

    git clone https://github.com/openvinotoolkit/model_server.git added

commit f681c0a1bd0fde212e8c4182d30bb42c9e84905e
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Mon Jun 20 10:50:47 2022 +0200

    docs/accelerators.md updated

commit a0fe6d543aaa66d59611a560ba055e9e26ebb324
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Fri Jun 17 16:36:32 2022 +0200

    docs/performance_tuning.md + docs/accelerators.md updated

commit 62b7e203b850a580d684d769bb0fcae31d8ad999
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Fri Jun 17 15:56:22 2022 +0200

    docs/accelerators.md updated

commit 6100e8835aaa4c1674527fac727ea290e03097b3
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 15 16:29:12 2022 +0200

    demos/image_classification/go/README.md updated

commit 1d74d23dc76917a6fc64f021830fd5bb92ee07da
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 15 16:11:08 2022 +0200

    docs/developer_guide.md make docker_build NO_DOCKER_CACHE=true removed from commands

commit 470d7d8cdda19ab2ba2cd4554ae1130a9a4bcd0c
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 15 16:09:36 2022 +0200

    demos/image_classification/go/README.md updated

commit a97ce3e48e8f15a8eeb53285cff368e1133b230c
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 15 15:50:38 2022 +0200

    demos/image_classification/go/README.md updated

commit e840b925b3869796976ec97c01835d456bc9c79e
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 15 09:39:46 2022 +0200

    client/python/ovmsclient/samples/utils/resnet_utils.py restored to 4ad1280 version

commit 539b5ff1edc69b949cbc9e0770927cd500d5985d
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Tue Jun 14 15:10:58 2022 +0200

    docs/stateful_models.md updated

commit aebd89c742be83892015836d388b1e6b83815025
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Tue Jun 14 15:10:29 2022 +0200

    docs/single_model_mode.md updated

commit 5ac27ffb9089753d2f5ce10861a62b512fe160fe
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Tue Jun 14 15:10:12 2022 +0200

    docs/performance_tuning.md updated

commit 7ffaa953660985a2c0677d120680f3b59a452dee
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Tue Jun 14 15:09:40 2022 +0200

    docs/accelerators.md updated

commit dfe905c86e15e348f5dd615a5934ebcc743a6ac7
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Tue Jun 14 15:09:21 2022 +0200

    docs/developer_guide.md updated

commit 6088e1b2c187c675a4eef70068ca6c5af7a690d9
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Tue Jun 14 15:02:49 2022 +0200

    demos/model_ensemble/python/README.md paths updated

commit 513a5ef5205993629869af7baa11ba03e0f803dd
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Tue Jun 14 15:01:28 2022 +0200

    demos/image_classification/go/README.md models added

commit 50b82a49e87e7e4c6d477feec2e0ead67b8b4335
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Tue Jun 14 15:00:04 2022 +0200

    demos/face_detection/python/README.md output added

commit f97d37ca31ab9e2ba45b279988fd0df25d6bd72d
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Tue Jun 14 14:59:27 2022 +0200

    demos/benchmark/python/README.md output reverted

commit cc8aa9afc4c422286541cfe708fca11dccea6a35
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Tue Jun 14 14:58:22 2022 +0200

    docs/using_cloud_storage.md bash tags added

commit b12750e4aa3cfb45672ac94c816b4d16171666b1
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Fri Jun 10 17:36:13 2022 +0200

    docs/dynamic_bs_demultiplexer.md Bash->bash

commit 5cfac292f89fdd4a95962a8c3eeca70d96edecef
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Fri Jun 10 17:35:35 2022 +0200

    docs/host.md Bash->bash

commit a67fe50e6ddef9712ea55b0fba3e1998b1644412
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Fri Jun 10 17:34:17 2022 +0200

    client/python/tensorflow-serving-api/samples/README.md Bash->bash

commit 3b5a6f9a2e49f2167ee517e6f3d0d7321c18dd7e
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Fri Jun 10 17:32:18 2022 +0200

    docs/tf_model_binary_input.md Bash->bash

commit ec53482134ac1c862a0b33a65b0810317581dcd1
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Fri Jun 10 17:15:25 2022 +0200

    client/python/ovmsclient/samples/README.md removed additional model download

commit a698cb8eed15aadb2d9bd2cbe4cfab293c3e0cb0
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Thu Jun 9 11:06:26 2022 +0200

    demos/benchmark/python/README.md reverted changed in output

commit fab9ade58b07aec1862faf3914dce93d1216b337
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 8 17:00:07 2022 +0200

    demos/optical_character_recognition/python/README.md description changed

commit cc7df438b9201e9c2f2e05523bf10fe3269ac535
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 8 16:56:45 2022 +0200

    docs/custom_model_loader.md description changed

commit b240a5b973a37cce5b5da9c0ad486b4a723f8464
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 8 16:51:50 2022 +0200

    docs/tf_model_binary_input.md git clone command added

commit b01ec84052e0581fc22f1e9a4b08f30d90d86fbe
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 8 16:42:33 2022 +0200

    docs/docker_container.md script.py -> predict.py

commit a7df7c2f25a2a2a423d7997f3d6cd5bc7505017d
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 8 16:41:45 2022 +0200

    docs/dynamic_shape_binary_inputs.md script.py -> predict.py

commit 84bb6ae61ead5fd9ea29a6eb09470df28ee0585e
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 8 15:49:19 2022 +0200

    demos/age_gender_recognition/python/README.md ## Download Sample Image header removed

commit ec0a710cf98eda17d6d186d6e4e4d765e262813e
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 8 11:57:18 2022 +0200

    demos/speech_recognition_with_kaldi_model/python/asr_demo/README.md updated

commit 797ee2ac1d6d36347cbd3b215b0ca0ab92a35191
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Tue Jun 7 14:38:51 2022 +0200

    demos/benchmark/python/README.md updated

commit fed9176a6f17013684d14bcc53f87d81eb0e4425
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Tue Jun 7 14:38:01 2022 +0200

    docs/docker_container.md updated

commit 4b95f16a3ab28f327689d082cdd64b7a15a63029
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Tue Jun 7 12:48:51 2022 +0200

    docs/model_cache.md updated

commit 44e11a02ea743021b47694389bb843e968b799cb
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Mon Jun 6 17:15:35 2022 +0200

    docs/multiple_models_mode.md updated

commit 86d86ec8fd717a8257b8cfaaf6dec75cde04ec78
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Mon Jun 6 17:14:13 2022 +0200

    docs/models_repository.md updated

commit c8da7ae85f40e429807e699e57a0fbcc9bccaeb9
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Mon Jun 6 17:07:24 2022 +0200

    docs/model_cache.md updated

commit 1e46d54aa13b1a61a83a44e7fd5d6c5fe487bf68
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Mon Jun 6 16:58:00 2022 +0200

    docs/model_cache.md updated

commit cd4aea1aeec7035d80533488049945e7fddae22f
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Mon Jun 6 15:09:23 2022 +0200

    docs/docker_container.md updated

commit 1c73a83877b85b53a28a1de7cbd528de74788cfd
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Mon Jun 6 15:08:50 2022 +0200

    docs/model_cache.md updated

commit 0c9113477c74c2f6f9e75c0227281b9beaca9cc4
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Mon Jun 6 15:08:23 2022 +0200

    docs/model_server_rest_api.md updated

commit 685c87cc42e270daf10b6d4d252d23451a9656ee
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Mon Jun 6 10:00:19 2022 +0200

    docs/docker_container.md updated

commit 2080dcb6aa591ee20f4652df65b6f8102251adf5
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Mon Jun 6 09:40:36 2022 +0200

    docs/dynamic_shape_binary_inputs.md expected output added

commit a9072b393ee8b4b6adb3ead130aed66697baa189
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Mon Jun 6 09:35:37 2022 +0200

    demos/bert_question_answering/python/README.md updated

commit 6f54ee19a4ef81805e073a013454714a8ce21661
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Mon Jun 6 09:34:24 2022 +0200

    demos/bert_question_answering/python/README.md reverted

commit 426f5940ea4cc71dab71472f8b2d50b09e6e2b32
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Fri Jun 3 15:38:34 2022 +0200

    demos/speech_recognition_with_kaldi_model/python/asr_demo/README.md updated

commit 9250af9a652a0d798299d4ec395d1181592ae88a
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Fri Jun 3 14:35:00 2022 +0200

     demos/speech_recognition_with_kaldi_model/python/asr_demo/README.md updated

commit a7db371f2c0909d824718dde05f45c4af3410b02
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Fri Jun 3 12:05:13 2022 +0200

    demos/image_classification/go/README.md updated

commit cde2c16b20485501b9f73ebed31afb1da8aa1f2f
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Fri Jun 3 12:03:44 2022 +0200

    client/python/tensorflow-serving-api/samples/README.md updated

commit 8877cc330399598aaf162231ac16f0247ab13953
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Fri Jun 3 10:12:27 2022 +0200

    demos/image_classification/go/README.md reverted

commit 8ba6fc91313ce1c15cb125ae465b2953a2f4dbf9
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Thu Jun 2 15:50:12 2022 +0200

    client/python/ovmsclient/samples/README.md updated

commit 423eb1b30bdc0ec7c761a14ccc7b83631981af0a
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Thu Jun 2 11:01:39 2022 +0200

    docs/dynamic_bs_demultiplexer.md updated

commit 8c22f8d69a24e8178704f1c8abec60ed7a51c0cc
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Thu Jun 2 10:26:12 2022 +0200

    docs/tf_model_binary_input.md updated

commit 7bdab4cb57dc85fce4f79e95f029ba7495a2702e
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Thu Jun 2 09:34:34 2022 +0200

    docs/ovms_quickstart.md updated

commit 74b2ed69d5943e167822b4cfe0c803ad08623a22
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Thu Jun 2 08:30:00 2022 +0200

    docs/tf_model_binary_input.md updated

commit 692e0f1ac90fbb1ecc2d598f287910c6d41dec3e
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Thu Jun 2 08:25:56 2022 +0200

    docs/troubleshooting.md updated

commit 2600c0c4def94bb295e5c10bfe93bfd8ae9cb720
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Thu Jun 2 08:07:55 2022 +0200

    docs/host.md revert

commit f397428fca11858e96ee56eee136bcc7ad290315
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 15:43:31 2022 +0200

    docs/custom_model_loader.md updated

commit 07a87a90fef26bc918bbc61856f9a119db88d083
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 15:43:02 2022 +0200

    docs/ bash added to code snippets

commit b1b7605239e82bc8c205866941d94e8ee32c8600
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 15:17:47 2022 +0200

    demos/model_ensemble/python/README.md updated

commit ae7be6e726fe6b1906a316672bc14eaada6e6910
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 15:16:03 2022 +0200

    demos/ bash added to code snippets

commit 780e439217140bb4635c3b0a409b78af5b105cd2
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 14:06:11 2022 +0200

    demos/age_gender_recognition/python/README.md updated

commit 832d43a23778e1ff87f7f1a2b44a3361ca8c4006
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 11:47:53 2022 +0200

    docs/dynamic_shape_dynamic_model.md updated

commit 5c3714b21ee40c5c0f2aa0bb2ef95a261c315fa3
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 11:47:33 2022 +0200

    docs/dynamic_shape_custom_node.md updated

commit afec4f5d466eed5fec2be51180b6631b9bcb79a2
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 11:46:47 2022 +0200

    docs/dynamic_shape_binary_inputs.md updated

commit 3f756edca120129293abafd9faffb7609f173083
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 11:45:45 2022 +0200

    docs/dynamic_shape_auto_reload.md updated

commit 9201ed50dda10a9d7c3895d65f6bfb42bd2709ad
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 11:45:09 2022 +0200

    docs/dynamic_bs_auto_reload.md output updated

commit c2cc9e1ac5ced343b0b3a9e29b24bbb43d808a8b
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 11:43:15 2022 +0200

    docs/binary_input.md updated

commit 82e871d26d7d6dc9ea13c3e28a752427900670a3
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 11:41:41 2022 +0200

     demos/speech_recognition_with_kaldi_model/python/README.md updated

commit dd22f4c558a233bc52bd0a2c71b8ad86033246e5
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 11:41:00 2022 +0200

     demos/speech_recognition_with_kaldi_model/python/README.md updated

commit 6854ee14f72ff221c4e9a414300873d1cabdcf41
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 11:39:04 2022 +0200

    demos/single_face_analysis_pipeline/python/README.md --grpc_port option added

commit eab1bd9dc12ef26f1be3ad7fe7856ca749b9b3c4
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 11:35:55 2022 +0200

    demos/real_time_stream_analysis/python/use_cases/person_vehicle_bike_detection/README.md updated

commit 0b7ef42fc8b5d4b91b27e34f7801d0113c8059cd
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 11:34:26 2022 +0200

    demos/real_time_stream_analysis/python/README.md updated

commit ab0b5ca9f7b8c7c78cc5af1ff45bfcf57c25668c
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 11:31:55 2022 +0200

    demos/person_vehicle_bike_detection/python/README.md output updated

commit 9b6886acfa94fc3d3150923bc47b64390b6b6e61
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 11:28:39 2022 +0200

    demos/bert_question_answering/python/README.md and demos/image_classification/go/README.md downloaded from git

commit b5064a7b8f6c75cb698bd578cd03ead668a0ae7c
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 11:19:45 2022 +0200

    demos/optical_character_recognition/python/README.md updated

commit 0ba2c286cdd4bd50d97ba063eade86316c3cb952
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 10:58:21 2022 +0200

    demos/image_classification/go/README.md updated

commit 826c6623e190650e473cb0839e51d3d27babba76
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 10:57:33 2022 +0200

    demos/image_classification/cpp/README.md make command added

commit 2bbb5d8e5ffdc316dd0d0913edc496e195943c59
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 10:53:11 2022 +0200

    demos/face_detection/python/README.md --grpc_port option added

commit 1b8a631752624d1eb57378664682c195b1213367
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 10:52:10 2022 +0200

    demos/face_blur/python/README.md tree output updated

commit 6c6d5530935e7e1df00edfd8fedd83cdb4ec2062
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 10:50:51 2022 +0200

    demos/bert_question_answering/python/README.md output updated

commit 4a0ab1667bcd8d7e9c616cba7243f7afbd2f3f07
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 08:54:32 2022 +0200

    demos/benchmark/cpp/README.md and demos/benchmark/python/README.md updated

commit 9e7602d6d60ddac602e6b92f46328a5b277107f1
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Wed Jun 1 08:41:10 2022 +0200

    demos/age_gender_recognition/python/README.md updated

commit 4745de6baf3a2e2080f0bcb2a2860c305f329acf
Author: Natalia Groza natalia.groza@intel.com <ngroza@ngroza-u20.sclab.intel.com>
Date:   Thu May 26 18:11:07 2022 +0200

    cvs-78730 model_ensemble/README.md update

### CODE REVIEW

File: client/python/ovmsclient/README.md [hunk 1]
```diff
@@ -18,4 +20,13 @@ If you're looking for a quick way to try `ovmsclient` out or want to look up how
 There are also Dockerfiles available that prepare Docker image with `ovmsclient` installed and ready to use.
 Simply run `docker build` with the Dockerfile of your choice to get the minimal image:
 - [Ubuntu 20.04 based image](Dockerfile.ubuntu)
-- [UBI 8.4 based image](Dockerfile.redhat)
\ No newline at end of file
+- [UBI 8.4 based image](Dockerfile.redhat)
```
Comment (mzegla) [inline client/python/ovmsclient/README.md:23]: I think we are using UBI 8.5 starting from that release.

File: demos/optical_character_recognition/python/README.md [hunk 2]
```diff
@@ -145,7 +164,6 @@ docker run -p 9000:9000 -d -v ${PWD}/OCR:/OCR openvino/model_server --config_pat
 
 Clone the repository and enter optical_character_recognition directory
```
Comment (mzegla) [inline demos/optical_character_recognition/python/README.md:165]:
```suggestion
Enter optical_character_recognition directory
```

File: demos/speech_recognition_with_kaldi_model/python/asr_demo/README.md [hunk 3]
```diff
@@ -178,15 +181,16 @@ Run the live-demo.py script to record and send audio files to the server:
 <SERVER_IP> - IP of the unix server with the running ovms and kaldi containers from steps 1 to 5.
 <SERVER_HOME_PATH> - is the path of the $HOME directory from steps 1 to 5.
 <SERVER_USER_NAME> - is the owner of the $HOME path and a user of the server used to run steps 1 to 5.
-```
-CLIENT SIDE:
-python live-demo.py <SERVER_IP> <SERVER_HOME_PATH>/asr_demo/data <SERVER_USER_NAME>
+```bash
+#CLIENT SIDE:
+#python live-demo.py <SERVER_IP> <SERVER_HOME_PATH>/asr_demo/data <SERVER_USER_NAME>
```
Comment (mzegla) [inline demos/speech_recognition_with_kaldi_model/python/asr_demo/README.md:186]: Can we remove that?

Comment (ngrozae) [reply demos/speech_recognition_with_kaldi_model/python/asr_demo/README.md]: Done

File: demos/optical_character_recognition/python/README.md (hunk 2 above)
Comment (ngrozae) [reply demos/optical_character_recognition/python/README.md]: Done

File: client/python/ovmsclient/README.md (hunk 1 above)
Comment (ngrozae) [reply client/python/ovmsclient/README.md:23]: from client/python/ovmsclient/Dockerfile.redhat (develop branch):
ARG BASE_IMAGE=registry.access.redhat.com/ubi8/ubi:8.4
ARG FINAL_IMAGE=registry.access.redhat.com/ubi8/ubi-minimal:8.4

**Review by mzegla — APPROVED** [review]
In another PR we should update UBI version in ovmsclient dockerfile to 8.5

**Review by michalkulakowski — APPROVED** [review]

**Pull request merged**